src/pages/mc-currency-mapping/index.js

```diff
@@ -1,4 +1,4 @@
-import { useState, useContext } from 'react'
+import { useContext } from 'react'
 import toast from 'react-hot-toast'
 import Table from 'src/components/Shared/Table'
```

Comment:
remove  const invalidate = useInvalidate({
    endpointId: MultiCurrencyRepository.McExchangeMap.page
  }).  from index and put invalidate in useResourceQuery